installer/windows/Product.wxs

```diff
@@ -60,11 +60,12 @@
           Arguments="run --config-dir &quot;[INSTALLFOLDER]\&quot; --identity-dir &quot;[IDENTITYDIR]\&quot; --binary-location &quot;[INSTALLFOLDER]\storagenode.exe&quot; --log &quot;[INSTALLFOLDER]\storagenode-updater.log&quot;">
             <util:ServiceConfig
               ServiceName="storagenode-updater"
-              RestartServiceDelayInSeconds="5"
+              ResetPeriodInDays="1"
+              RestartServiceDelayInSeconds="60"
               FirstFailureActionType="restart"
-              SecondFailureActionType="restart"
+              SecondFailureActionType="runCommand"
               ThirdFailureActionType="runCommand"
-              ProgramCommandLine="&quot;[INSTALLFOLDER]storagenode-updater.old.exe&quot; recover --config-dir &quot;[INSTALLFOLDER]\&quot; --log &quot;[INSTALLFOLDER]storagenode-updater.log&quot;"
+              ProgramCommandLine="cmd /k &quot;copy /y &quot;[INSTALLFOLDER]storagenode-updater.old.exe&quot; &quot;[INSTALLFOLDER]storagenode-updater.exe&quot; &amp;&amp; net start storagenode-updater&quot;"
```

Comment:
Is it possible to redirect the output of this command to the `storagenode-updater.log`?

Reply:
I need to experiment

Reply:
it should be possible to add something like that to log:
```
Fri 11/08/2019 11:15:41.39 recovery command executed: 
        1 file(s) copied.

Waiting for 5 seconds, press a key to continue ...43210
The Storj V3 Storage Node Updater service is starting.
The Storj V3 Storage Node Updater service was started successfully.
```
but I'm not sure how to suppress timeout second counting

Reply:
done

Reply:
Cool!
Can we also print something to the log before the timeout? Like "recovery command will be executed in 30 minutes"? This may give more clues to SNOs of what is going on during these 30 minutes.

Reply:
maybe we can but it will make this command much longer becaue I think we need to do something like this `(echo recovery command will be executed in 30 minutes >>  path_to_log) && rest of commands like right now...`